plugins/ruby-foreman-remote-execution/debian/ruby-foreman-remote-execution-cockpit.install

```diff
@@ -0,0 +1,4 @@
+extra/cockpit/foreman-cockpit-session usr/sbin/
+extra/cockpit/foreman-cockpit.service lib/systemd/system/
```

Comment:
The service file use `/usr/libexec/cockpit-ws`, while on Debian the executable is in `/usr/lib/cockpit/cockpit-ws`.

@adamruzicka @ekohl what do you think would be the best path here? Just patch the .service file in Debian packaging?

Reply:
I think `libexec` is more correct, but IIRC `libexec` isn't defined in the FHS. I think patching it on Debian is fine.

Reply:
At this point the FHS is pretty much dead, and Debian allows `libexec` in packages just fine. But given we're not packaging cockpit, we can't fix it

Reply:
I still think FHS is a good reference, but indeed it should receive an update that's probably not going to exist. Should we file a Debian bug and link to it when patching?

Reply:
I don't think it's necessary